agent/ansible/Makefile

```diff
@@ -11,14 +11,16 @@ version = $(shell ./bin/yamlconf -C ./pbench/agent/galaxy.yml version)
 tb = ${namespace}-${collection}-${version}.tar.gz
 colldir = collections/ansible_collections/${namespace}/${collection}
 
-build:
+lint:
+	make -C pbench/agent
```

Comment:
What does this `make` command do?  For instance, it's setting the current working directory to a subdirectory, but I have no idea what the _current_ CWD is, so I'm not sure where this is going to end up.

And, I can't find any `Makefile`'s in an `agent` subdirectory of a `pbench` directory:
```
$ find . -path "*/pbench/agent" -print -exec ls Makefile \;
./agent/ansible/pbench/agent
ls: Makefile: No such file or directory
./lib/pbench/agent
ls: Makefile: No such file or directory
./build/lib/pbench/agent
ls: Makefile: No such file or directory
```

And, I don't see such a `Makefile` in this PR...so, what am I missing?

Reply:
Yes, curious about this addition.

Would it be better to move the ansible playbooks to an entirely new git repo?

Reply:
@webbnh I think  both are "wrong" :  `test` should be the default target. That's the one I've used most often - well, `publish` is probably the one I used most often, but that's only because Galaxy is so picky - I don't think I want to make it the default.